README.md

```diff
@@ -126,6 +126,60 @@ func main() {
 }
 ```
 
+## Pagination
+
+Use the `Client.Paginate()` method to iterate sets that contain more than one page of results.
+
+`Client.paginate()` accepts the same query options as `Client.Query()`.
+
+Change the default items per page using FQL's `<set>.pageSize()` method.
+
+```go
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fauna/fauna-go"
+)
+
+type Product struct {
+	Description string `fauna:"description"`
+}
+
+func main() {
+	client, clientErr := fauna.NewDefaultClient()
+	if clientErr != nil {
+		panic(clientErr)
+	}
+
+	// Adjust `pageSize()` size as needed.
+	query, _ := fauna.FQL(`
+    Product
+      .byName("limes")
+      .pageSize(2) { description }`, nil)
+
+	options := fauna.Timeout(time.Minute)
+
+	paginator := client.Paginate(query, options)
```

Comment:
Is this necessary or can we pass nil and get default behaviors?

Reply:
Not necessary. Was just showing how you _could_ pass options. Removed with [`6b75450` (#136)](https://github.com/fauna/fauna-go/pull/136/commits/6b7545076461452b379ab1cd1d542a276064b265)

Reply:
Thanks!